flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/AvroRowDataDeSerializationSchemaTest.java

```diff
@@ -198,4 +198,38 @@ public void testSpecificType() throws Exception {
 		Assert.assertEquals("12:12:12", DataFormatConverters.LocalTimeConverter.INSTANCE.toExternal(
 				rowData.getInt(2)).toString());
 	}
+
+	@Test
+	public void testNullableMapType() throws Exception {
```

Comment:
We don't need to add such a big test. You can just add a null entry to the `map` in the `testSerializeDeserialize`.

Reply:
OK, i will remove it , and add in the `testSerializeDeserialize`